CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift

```diff
@@ -87,10 +104,14 @@ class OnboardingViewController: UIViewController {
                 layoutBottomNavigationBar(navigationBar)
             }
         } else {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Skip",
-                                                                style: .plain,
-                                                                target: self,
-                                                                action: #selector(close))
+            nextButton.layer.cornerRadius = 14
+            nextButton.setTitle("Next", for: .normal)
```

Comment:
[Suggestion] I think we already have this translated

Reply:
yes, the PR just includes UI updates. Texts will be handled in separate task, PR